CMakeLists.txt

```diff
@@ -57,6 +57,20 @@ include(GNUInstallDirs)
 add_subdirectory(src)
 add_subdirectory(dist)
 
+# Generate version header
+set(QBT_PROJECT_VERSION "${qBittorrent_VERSION_MAJOR}.${qBittorrent_VERSION_MINOR}.${qBittorrent_VERSION_PATCH}")
+if (NOT ${qBittorrent_VERSION_TWEAK} EQUAL 0)
+    set(QBT_PROJECT_VERSION "${QBT_PROJECT_VERSION}.${qBittorrent_VERSION_TWEAK}")
+endif()
+
```

Comment:
Seems you forgot to add QBT_VER_STATUS to QBT_PROJECT_VERSION.

Reply:
Thanks, fixed.